src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -310,6 +311,17 @@ public boolean isProperty(String name) {
         return options.javafx() && name.endsWith("Property");
     }
 
+    /*
+     * If a similar query is added to javax.lang.model, use that instead.
```

Comment:
The `Trees` API would be a better place than `java.lang.model` since this is just a query on an AST node, not on an `Element`.